src/pages/send-tokens/send-tokens.tsx

```diff
@@ -2,7 +2,7 @@ import React, { memo, useCallback, useState } from 'react';
 import { Box, Text, Button, Stack } from '@stacks/ui';
 import { Formik, FormikProps } from 'formik';
 import { PopupContainer } from '@components/popup/container';
-import { ConfirmSendDrawer } from '@components/drawer/confirm-send-drawer';
+import { ConfirmSendDrawer } from '@pages/transaction-signing/components/confirm-send-drawer';
```

Comment:
this should not be in `pages/transaction-signing/components`, it's shared between 2 features/pages